constants/links.js

```diff
@@ -1 +1 @@
-export default [{ title: 'Upay', url: '/' }, { title: 'About', url: '/about' }];
+export default [{ title: 'Upay', url: '/' }, { title: 'About', url: '/about' } , {title: 'Login', url:'/login'}];
```

Comment:
Hi!

It seems you've not configured prettier with your editor. Please read the README.md and set it up that way. Also I'll recommend having it `formatOnSave`.

Reply:
I'll run the code on prettier. Shall I remove reactstrap also? Although it makes it easy for the responsiveness.

Reply:
Yup.

That's really good but we have flex boxes, css grid and media queries in 2018. We don't really need bootstrap like we used to back in the day.

Also, this way you'll learn how these frameworks actually work internally. 

I feel that GSSoC is the opportunity where you learn and code, write stuff from scratch and actually evolve. And that's why I'm imposing such weird rules. I hope you understand and I really appreciate your patience with my review comments. Thank you so much!

Reply:
I will go ahead without reactstrap then. Thanks for the guidance!

Reply:
Great. As a side note, try to create an issue first, get views of fellow mentors and participants, claim it and then start working. I'm saying this to avoid working in a direction that might be slightly different from what product owners might want.

Reply:
Sure! I will always keep that in mind.
So shall I wait for the mockup designs as of now?

Reply:
Yeah I guess. Just use bareborne CSS for now, we can rework it always.

I feel we can add simple UI components for now, like 

* Button
* Add button
* Save for later button
* Checkout button
* Drop down list
* Categories list
* Item (image title description price)
* Item list (with sort and filter)
* Search bar
* Carousel
* Cart
* Modal

Reply:
By bareborne CSS do you mean minimal CSS?

Reply:
Yup, not too much effort on design as it's bound to change.

Reply:
Could you do a favor and create individual issues for about mentioned components? You can claim whichever ones you like.

Reply:
Sure! I will do that. You meant issues for all the above-mentioned components like Cart, , Modal and all, right?

Reply:
Yup. Preferably with some description or sample image for new comers to understand what is being asked for here.

Reply:
Thank you so much!

Reply:
Can you please check for the description of the issues that I am posting? If its fine then I will open more issues for the rest of the common components.

Reply:
You can rename title as `Add <BuyButton />` to be easy to read.